cmd/faucet/pkg/config/config.go

```diff
@@ -0,0 +1,85 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/caarlos0/env/v10"
+	"github.com/spf13/viper"
+)
+
+var errConfig = errors.New("config error")
+
+func configError(msg string) error {
+	return fmt.Errorf("%w: %s", errConfig, msg)
+}
+
+type Config struct {
+	Port           string `env:"PORT" envDefault:"8081" mapstructure:"PORT"`
+	EnvFile        string `env:"ENV_FILE" envDefault:""`
+	PurgeInterval  string `env:"PURGE_INTERVAL" envDefault:"10s" mapstructure:"PURGE_INTERVAL"`
+	Mnemonic       string `env:"MNEMONIC" envDefault:"" mapstructure:"MNEMONIC"`
+	Node           string `env:"NODE" envDefault:"https://rpc.buenavista.wardenprotocol.org:443" mapstructure:"NODE"`
+	ChainID        string `env:"CHAIN_ID" envDefault:"buenavista-1" mapstructure:"CHAIN_ID"`
+	CliName        string `env:"CLI_NAME" envDefault:"wardend" mapstructure:"CLI_NAME"`
+	AccountName    string `env:"ACCOUNT_NAME" envDefault:"faucet" mapstructure:"ACCOUNT_NAME"`
+	Denom          string `env:"DENOM" envDefault:"uward" mapstructure:"DENOM"`
+	Amount         string `env:"AMOUNT" envDefault:"1000000" mapstructure:"AMOUNT"`
+	Fees           string `env:"FEES" envDefault:"25uward" mapstructure:"FEES"`
+	CoolDown       string `env:"COOLDOWN" envDefault:"10s" mapstructure:"COOLDOWN"`
```

Comment:
can this be a time.Duration?

Reply:
We don't actually need the cooldown for this